pyanaconda/storage/utils.py

```diff
@@ -520,9 +520,21 @@ def filter_disks_by_names(disks, names):
 
 
 def is_local_disk(disk):
-    return (not isinstance(disk, MultipathDevice)
-            and not isinstance(disk, iScsiDiskDevice)
-            and not isinstance(disk, FcoeDiskDevice))
+    """Is the disk local?
+
+    A local disk doesn't require any additional setup unlike
+    the advanced storage.
+
+    While technically local disks, zFCP and NVDIMM devices are
+    specialized storage and should not be considered local.
+
+    :param disk: a disk
```

Comment:
Please change `a disk` to something like `a disk which inherit from a blivet.devices.device.Device`.

Reply:
Fixed.